webrender/res/prim_shared.glsl

```diff
@@ -548,9 +539,11 @@ vec4 get_layer_pos(vec2 pos, Layer layer) {
     // get a point on the layer plane
     vec4 ah = layer.transform * vec4(0.0, 0.0, 0.0, 1.0);
     vec3 a = ah.xyz / ah.w;
+
     // get the normal to the layer plane
-    vec3 n = transpose(mat3(layer.inv_transform)) * vec3(0.0, 0.0, 1.0);
-    return untransform(pos, n, a, layer.inv_transform);
+    mat4 inv_transform = inverse(layer.transform);
```

Comment:
I don't think it's a good idea to compute the inverse mat4 per vertex.

Reply:
So I definitely want to do some more testing here, but I believe this is only called for clips. In any case, it's probably pretty easy to move this back to the CPU only for nodes that need it.